core/modules/system/system.module

```diff
@@ -2433,8 +2433,10 @@ function system_block_info() {
   foreach (menu_list_system_menus() as $menu_name => $title) {
     $custom_title = isset($all_menus[$menu_name]) ? $all_menus[$menu_name] : $title;
     $blocks[$menu_name] = array(
+      // Keeping $custom_title wrapped in t(), to allow menu names to be
+      // translated.
```

Comment:
This comment seems like it's speaking to this PR. I would say remove it or reword to something like:
```
Names from menu_list_system_menus() need to be translated through t().
```

Reply:
Done! 😄